regression-tests.dnsdist/test_Dnstap.py

```diff
@@ -512,3 +514,96 @@ def testDnstapOverFrameStreamTcp(self):
 
         checkDnstapQuery(self, dnstap, dnstap_pb2.UDP, query)
         checkDnstapNoExtra(self, dnstap)
+
+class TestDnstapOverRemotePoolTcpLogger(DNSDistTest):
+    _fstrmLoggerPort = pickAvailablePort()
+    _fstrmLoggerQueue = Queue()
+    _fstrmLoggerCounter = 0
+    _poolConnectionCount = 5
+    _config_params = ['_testServerPort', '_fstrmLoggerPort', '_poolConnectionCount']
+    _config_template = """
+    newServer{address="127.0.0.1:%s", useClientSubnet=true}
+    fslu = newFrameStreamTcpLogger('127.0.0.1:%s', { connectionCount = %s })
```

Comment:
It doesn't really matter but this is a bit cleaner:
```suggestion
    newServer{address="127.0.0.1:%d", useClientSubnet=true}
    fslu = newFrameStreamTcpLogger('127.0.0.1:%d', { connectionCount = %d })
```